test/config.ts

```diff
@@ -48,3 +48,16 @@ export const getAddresses = () => {
   )
   return data.development
 }
+
+export const getAddressesForSapphire = (testnet: boolean) => {
```

Comment:
Maybe it's a good ideea to have two functions:
 - getAddressesForChain(chainID)
 - getAddressesForSapphire which calls getAddressesForChain

Reply:
I have that locally, not pushed yet.